xspec/.codespellrc

```diff
@@ -0,0 +1,8 @@
+[codespell]
```

Comment:
why do we need these files? 
Please delete the 4 `.xxx` files. In the future these shall be (Git) ignored if necessary only in the dev environment.

Reply:
do we need XSPEC folder? 
I start doubting this. What is its purpose?